VologramsToolkit/Scripts/Player/VolPlayer.cs

```diff
@@ -106,26 +107,39 @@ private void Update()
     {
         if (!IsPlaying) return;

```

Comment:
this is the meat and potatoes of this PR - using a timer to work out the appropriate frame to be at, and always skipping to that, rather than going one by one